src/test/resources/net/sf/jabref/logic/exporter/ModsExportFormatTestMultipleEntries.bib

```diff
@@ -0,0 +1,20 @@
+% Encoding: UTF-8
+
+@Article{,
+  author   = {Shruthi, Basavaradhya Sahukar and Vinodhkumar, Palani and Selvamani},
+  title    = {Proteomics: A new perspective for cancer.},
+  journal  = {Advanced biomedical research},
```

Comment:
Btw, the shemas are now here on github, too: according to their release annoucncement:
http://www.loc.gov/standards/mods/mods-3-6-announcement.html